src/components/Page/components/Header/Header.tsx

```diff
@@ -35,23 +35,21 @@ export interface PrimaryActionProps
 export interface Props {
   /** Page title, in large type */
   title: string;
-  /** Important and non-interactive status information shown immediately after the title. */
+  /** Important and non-interactive status information shown immediately after the title. (stand-alone app use only) */
   titleMetadata?: React.ReactNode;
-  /** Visually hide the title */
+  /** Visually hide the title (stand-alone app use only) */
   titleHidden?: boolean;
-  /** App icon, for pages that are part of Shopify apps */
-  icon?: string;
```

Comment:
This was 🔥 in `3.0.0` but we never removed the prop from this interface.

Reply:
Good catch

Reply:
As discussed, we should keep this prop and instead add a console warning that the prop no longer does anything and to use the Partners dashboard to upload your icon.